resources/webdid/test-dataspace-authority/did.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
nit: how about we reused the existing did json for this ? Not 100% sure on this but we can discuss. Just a suggestion.

Reply:
After thinking about it I would leave it in 2 separate files. It doesn't cost much more and it helps to avoid the confusion why the client and the dataspace authority have the same did (even if it would be only for testing purposes).
But happy to hear other idea